datalad_fuse/tests/test_fsspec_head.py

```diff
@@ -0,0 +1,87 @@
+# datalad.api needs to be imported in order for fsspec_head to be available
+import datalad.api  # noqa
+from datalad.tests.utils import assert_in_results
+
+
+def test_get_default_lines_text(url_dataset):
+    assert_in_results(
+        url_dataset.fsspec_head("text.txt"),
+        action="fsspec-head",
+        type="dataset",
+        status="ok",
+        data=b"""\
+Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh
```

Comment:
do we really need "hardcoded" sample binary/text files/varbatim copy in the code here?  Why not just to use some `random` data if the actual value does not matter and not to bother doing all this script flooding with the "explicit" target value? 

may be even worth using https://pypi.org/project/hypothesis/ here and let it sweep for us through possible "interesting" values

Reply:
@yarikoptic I don't see a problem with the current method.